Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -434,6 +434,8 @@ final class SyncChangeStreamTests: MongoSwiftTestCase {
      */
     func testChangeStreamAutomaticResume() throws {
         let testRequirements = TestRequirement(
+            // TODO SWIFT-1257: remove server version requirement
+            maxServerVersion: ServerVersion(major: 4, minor: 9, patch: 0),
```

Comment:
technically 5.0 is fine, the problem is just on latest, however as you recently noted in driver-devs the version numbers are currently wrong for the server's master branch so I think we just have to skip on 5.0 too